src/transforms/normalize.js

```diff
@@ -61,6 +64,18 @@ const normalizeLast = normalizeBasis((I, S) => {
   }
 });
 
+const normalizeDeviation = {
+  map(I, S, T) {
+    const m = mean(I, i => S[i]);
+    const d = deviation(I, i => S[i]);
+    for (const i of I) {
+      T[i] = S[i] === null ? NaN : (S[i] - m) / d;
+    }
+  }
+};
+
+const normalizeMax = normalizeBasis((I, S) => max(I, i => S[i]));
 const normalizeMean = normalizeBasis((I, S) => mean(I, i => S[i]));
 const normalizeMedian = normalizeBasis((I, S) => median(I, i => S[i]));
-const normalizeSum = normalizeBasis((I, S) => sum(I, i => S[i]));
+const normalizeMin = normalizeBasis((I, S) => min(I, i => S[i]));
+const normalizeSum = normalizeBasis((I, S) => sum(I, i => S[i]));
```

Comment:
Please preserve the trailing newline at the end of this file. 🙏

Reply:
Updated (and preserved!). Let me know if you prefer a different phrasing or ordering (I left them as alphabetical, with `extent` and `deviation` at the end, as they follow a different pattern than those above)